bin/gen-crons.py

```diff
@@ -23,6 +23,8 @@ def main():
                             'Only define for cron.d style crontabs.'))
     parser.add_option('-p', '--python', default='python2.6',
                       help='Python interpreter to use.')
+    parser.add_option('-n', '--nologdir', action='store_true',
```

Comment:
*HOWEVER*. We could in deployment pass in a `-l /var/log/bedrock` and that would do the check if provided. We could just not output the log stuff at all if it's not.